core/src/main/java/com/opensymphony/xwork2/interceptor/ChainingInterceptor.java

```diff
@@ -160,7 +160,20 @@ private void copyStack(ActionInvocation invocation, CompoundRoot root) {
         Map<String, Object> ctxMap = invocation.getInvocationContext().getContextMap();
         for (Object object : list) {
             if (shouldCopy(object)) {
-                reflectionProvider.copy(object, invocation.getAction(), ctxMap, prepareExcludes(), includes);
+                String actionConfigClassName = invocation.getProxy().getConfig().getClassName();
```

Comment:
As I guessed, this line does not work when action is a spring bean (`getClassName` returns the spring bean id, not fully qualified name of class).

Fortunately it works with actions created by convention plugin.

I will be working on it by checking all references which set or get class name. I also will add some unit tests to check if similar issues occurs in future.